src/test/scala/mesosphere/marathon/core/task/update/impl/steps/ScaleAppUpdateStepImplTest.scala

```diff
@@ -6,133 +6,88 @@ import com.google.inject.Provider
 import mesosphere.UnitTest
 import mesosphere.marathon.MarathonSchedulerActor.ScaleRunSpec
 import mesosphere.marathon.core.condition.Condition
+import mesosphere.marathon.core.condition.Condition._
 import mesosphere.marathon.core.event.MarathonEvent
-import mesosphere.marathon.core.instance.update.InstanceUpdated
+import mesosphere.marathon.core.instance.update.{InstanceChange, InstanceUpdated}
 import mesosphere.marathon.core.instance.{Goal, Instance, TestInstanceBuilder}
 import mesosphere.marathon.state.{PathId, Timestamp}
 
 class ScaleAppUpdateStepImplTest extends UnitTest {
 
-  // used pattern matching because of compiler checks, when additional case objects are added to Condition
-  def scalingWorthy: Condition => Boolean = {
-    case Condition.Reserved | Condition.Created | Condition.Killing | Condition.Running |
-      Condition.Staging | Condition.Starting | Condition.Unreachable => false
-    case Condition.Error | Condition.Failed | Condition.Finished | Condition.Killed |
-      Condition.UnreachableInactive | Condition.Gone | Condition.Dropped | Condition.Unknown => true
-  }
-
   val allConditions = Seq(
-    Condition.Reserved,
-    Condition.Created,
-    Condition.Error,
-    Condition.Failed,
-    Condition.Finished,
-    Condition.Killed,
-    Condition.Killing,
-    Condition.Running,
-    Condition.Staging,
-    Condition.Starting,
-    Condition.Unreachable,
-    Condition.UnreachableInactive,
-    Condition.Gone,
-    Condition.Dropped,
-    Condition.Unknown
+    Reserved,
+    Created,
+    Scheduled,
+    Error,
+    Failed,
+    Finished,
+    Killed,
+    Killing,
+    Running,
+    Staging,
+    Starting,
+    Unreachable,
+    UnreachableInactive,
+    Gone,
+    Dropped,
+    Unknown
```

Comment:
I'm not sure what the value of this diff is except for a star import.

Reply:
I think it drops some test code that is not needed anymore, because scale check is not triggered for some cases now...

Reply:
But why the wildcard import?